tools/firtool/firtool.cpp

```diff
@@ -128,6 +128,12 @@ static cl::opt<std::string>
     replSeqMemFile("repl-seq-mem-file",
                    cl::desc("file name for seq mem metadata"), cl::init(""));
 
+static cl::opt<bool>
+    ignoreReadEnableMem("ignore-read-enable-mem",
```

Comment:
Is there a way to plumb pass options up through firtool more generically (similar to how circt-opt gets them) instead of having to add each as top level flags?

Reply:
Tried to look around, seems like the way firtool is implemented, it is parsing its own command line arguments, so we have to add the option as top level flags. 
Maybe we need to redesign firtool to reuse the pass options.